serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,16 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel. Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a Subscription.
+
+The default configuration for channel instances is defined in a ConfigMap by the cluster administrator, however developers can still create their own channels directly by instantiating one of the supported channel objects.
+// InMemoryChannel, what other objects do we support OOTB? Kafka?
```

Comment:
In TP, we support the `InMemoryChannel`

the KafkaChannel (from the Kafka operator) is still DevPreview